embc-app/DataInterfaces/DataInterface.Organization.cs

```diff
@@ -28,8 +28,8 @@ public async Task<IPagedResults<Organization>> GetOrganizationsAsync(SearchQuery
                 regionName = searchQuery.Query;
             }
             var items = await Organizations
-                .Where(o => (!communityId.HasValue || o.Community.Id == communityId) ||
-                    (string.IsNullOrEmpty(regionName) || o.RegionName.Equals(regionName, StringComparison.OrdinalIgnoreCase))
+                .Where(o => (!communityId.HasValue || o.Community.Id == communityId) &&
+                    (string.IsNullOrEmpty(regionName) || o.RegionName == regionName)

```

Comment:
Not worried about case sensitivity?

Reply:
Sql Server is case insensitive, at least the way we configured our DB collation. I tested locally and any region name casing worked